docs/KnownIssues.md

```diff
@@ -0,0 +1,27 @@
+# Known Issues
+
+This page lists all the ongoing known issues in PowerApps-Language-Tooling for which we have workarounds. This document will be continuously updated as issues and workarounds are discovered.
```

Comment:
It could be helpful to have a little more detail here around the root of these issues. These don't necessarily represent "bugs we won't fix" as much as they represent changes that have been made in the underlying .msapp format which can cause roundtrip errors when unpacking or packing an app. These msapps can be brought up to the latest version, which will not encounter a roundtrip error, through the workarounds below.